sonic_platform_base/sonic_xcvr/codes/public/sff8436.py

```diff
@@ -0,0 +1,106 @@
+from .sff8024 import Sff8024
```

Comment:
Are you planning to reuse these codes in SFF 8636 because all below codes are available in SFF 8636 also.

Reply:
Good point, I had them separate but it might make sense to reuse them.